tests/cover/test_reflection.py

```diff
@@ -520,15 +520,15 @@ def test_copy_argspec_validates_function_name():
 
 class Container(object):
 
-    def funcy(self):
+    def function(self):
```

Comment:
I'm happy to use `func` which is a fairly standard choice. I'm also happy to drop this change or any others.